moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua

```diff
@@ -162,6 +162,10 @@ schemas["testpilottest"]                        = schemas.vacuous
 schemas["malware-addon-states"]                 = schemas.vacuous
 schemas["sync"]                                 = schemas.vacuous
 schemas["outofdate-notifications-system-addon"] = schemas.vacuous
+schemas["x-shield-studies"]                     = schemas.vacuous
+schemas["shield-study"]                         = schemas.vacuous
+schemas["x-contextual-feature-recommendation"]  = schemas.vacuous
+schemas["FirefoxOS"]                            = schemas.vacuous
```

Comment:
Should we just default to using the vacuous schema for any type where we do not have a specific schema?